examples/index.html

```diff
@@ -638,7 +643,7 @@ <h3>Output</h3>
   // set filters
   $('.set-filters').on('click', function() {
     $(this).prop('disabled', true);
-    $(this).tooltip('hide');
+    // $(this).tooltip('hide');
```

Comment:
is this still needed?

the following may work if you were looking to convert `bootstrap.Tooltip.getInstance($(this)).hide();`